internal/node/machine_mgr.go

```diff
@@ -587,14 +615,20 @@ func (m *MachineManager) generateTriggerHandler(vm *runningFirecracker, tsub str
 			m.t.functionRunTimeNano.Add(m.ctx, runTimeNs64, metric.WithAttributes(attribute.String("namespace", vm.namespace)))
 			m.t.functionRunTimeNano.Add(m.ctx, runTimeNs64, metric.WithAttributes(attribute.String("workload_name", *vm.deployRequest.WorkloadName)))
 
-			err = msg.Respond(resp.Data)
-			if err != nil {
-				m.log.Error("Failed to respond to trigger subject subscription request for deployed workload",
-					slog.String("vmid", vm.vmmID),
-					slog.String("trigger_subject", tsub),
-					slog.String("workload_type", *request.WorkloadType),
-					slog.Any("err", err),
-				)
+			if len(resp.Data) > 0 {
```

Comment:
This check for `if len(resp.Data) > 0` went away when we merged #127. We should be responding here even if data is empty.

Reply:
ugh, must've brought that back when I did a merge fix. I shall removeth